scripts.js

```diff
@@ -14,10 +16,112 @@ function createTileObject(name, top, left) {
   newTile.innerText = name;
   newTile.style.top = top + 'px';
   newTile.style.left = left + 'px';
+  newTile.id = Math.floor(Math.random() * 1000);
+  newTile.addEventListener('click', function () {
+    moveTile(this.id, this.parentElement);
+  });
+  newTile.draggable = 'true';
+  newTile.addEventListener('dragstart', function (ev) {
+    ev.dataTransfer.setData('div', ev.target.id);
+  });
+
+  tilesFirstContainer.push(newTile);
 
   return newTile;
 }
 
-function addTile() {}
+function addTile() {
+  const input = document.getElementById('tile-name').value;
```

Comment:
I am able to create empty tiles. I think we should put some limitations on the name. Think if there are any other checks you'd like to have around the naming.